### ISSUE TITLE
Stroke around massesControlPanel isn't instrumented

### ISSUE THREAD

**Issue by arouinfar** [phetsims/gravity-and-orbits#324]
Noticed during #321 

When setting `gravityAndOrbits.modelScreen.view.massesControlPanel.visibleProperty` or `gravityAndOrbits.toScaleScreen.view.massesControlPanel.visibleProperty` to false, the contents of the panel disappear but its stroke remains. This seems a bit buggy to me.

![image](https://user-images.githubusercontent.com/8419308/80326267-6c2f5500-87f5-11ea-91bb-5180354c2d56.png)

@samreid can you hide the stroke when the panel's top-level visibleProperty is false?

**Comment by samreid** [comment]
Good catch.  In the commit, I instrumented the outer Panel which contains the stroke rather than just the contents of the Panel.  While working on that, I checked to see if `gravityAndOrbits.modelScreen.view.controlPanel` had the same problem, and discovered that it was not instrumented at all (it was impossible to hide the entire top control panel).  I instrumented it using the same pattern.  Ready for review.
